Bottom: f3b752460246aa90151bb505907538bff3632b31
Top:    863391c75ceea20de1d2eb8a4e385e7077711af3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-12 01:00:27 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index b2add35..ff94ccf 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
